### PR TITLE
Update New-AzureRmOperationalInsightsWorkspace.md

### PR DESCRIPTION
**Pull request by MGoedtel**
Updated the SKU parameter to reflect the new pricing tier released this month called PerGB2018.

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md [hunk 1]
```diff
@@ -159,12 +160,13 @@ Valid values are:
...
 - standard
 - premium
+- PerGB2018
 
 ```yaml
 Type: String
 Parameter Sets: (All)
 Aliases:
-Accepted values: free, standard, premium, pernode, standalone
+Accepted values: free, standard, premium, pernode, standalone, PerGB2018
```
Comment (cormacpayne) [inline src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md]: @MGoedtel the `Sku` parameter has a [`ValidateSet`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Workspaces/NewAzureOperationalInsightsWorkspaceCommand.cs#L45), which means it only accepts values listed in the provided array (and these values above are generated from this list). We should ensure that the version of the SDK being used for OperationalInsights supports the `PerGB2018` value before adding it to the list of accepted values.

@haitch would you mind checking this out?

Comment (haitch) [reply src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md]: @cormacpayne I am no longer working on this project anymore.

Comment (cormacpayne) [reply src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md]: @xizhamsft would you mind taking a look?

Comment (MGoedtel) [reply src/ResourceManager/OperationalInsights/Commands.OperationalInsights/help/New-AzureRmOperationalInsightsWorkspace.md]: @cormacpayne - Understood and standing by.

**Comment by cormacpayne** [comment]
@xizhamsft gentle ping

**Comment by cormacpayne** [comment]
@MGoedtel Hey Matt, the new owners of this module said that the new value won't be found in the current version of the Operational Insights SDK being consumed by PowerShell, so it would be incorrect to update the documentation to reflect this. The owners also told me they have a task to update PowerShell to consume the latest version of the SDK reflected in the corresponding Swagger. As a result, I'm going to close this PR for now until the proper changes are made by the team.

**Comment by MGoedtel** [comment]
@cormacpayne - Understood.  I'll update our bug and would like to know if they provided an ETA?

**Pull request closed without merging**